### PR TITLE
More Workflow release notes 

### PR DESCRIPTION
**Pull request by nathanwoulfe**
## Description

Release notes for 10.3.8, 12.2.5 and 13.0.5

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Workflow 10, 12, 13

## Deadline (if relevant)

Whenever suits you :)

### CODE REVIEW

**Comment by alina-tincas** [comment]
Hi @nathanwoulfe thank you for the PR 😊 

We will review it as soon as we can 🙌

File: 10/umbraco-workflow/release-notes.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ If you are migrating from Plumber to Umbraco Workflow, see the [Migrate from Plu
 
 This section contains the release notes for Umbraco Workflow 10 including all changes for this version.
 
+### [10.3.8](https://github.com/umbraco/Umbraco.Workflow.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F10.3.8) (March 13th 2024)
+
+* Make dashboard, content app and section registration classes public [#56](https://github.com/umbraco/Umbraco.Workflow.Issues/issues/56)
```
Comment (eshanrnh) [inline 10/umbraco-workflow/release-notes.md:22]:
```suggestion
* Make dashboard, content app, and section registration classes public [#56](https://github.com/umbraco/Umbraco.Workflow.Issues/issues/56)
```

File: 12/umbraco-workflow/release-notes.md [hunk 2]
```diff
@@ -17,6 +17,11 @@ Check the [Version Specific Upgrade Notes](./upgrading/version-specific.md) arti
 
 This section contains the release notes for Umbraco Workflow 12 including all changes for this version.
 
+### [12.2.5](https://github.com/umbraco/Umbraco.Workflow.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F12.2.5) (March 13th 2024)
+
+* Make dashboard, content app and section registration classes public [#56](https://github.com/umbraco/Umbraco.Workflow.Issues/issues/56)
```
Comment (eshanrnh) [inline 12/umbraco-workflow/release-notes.md:22]:
```suggestion
* Make dashboard, content app, and section registration classes public [#56](https://github.com/umbraco/Umbraco.Workflow.Issues/issues/56)
```

File: 13/umbraco-workflow/release-notes.md [hunk 3]
```diff
@@ -17,6 +17,11 @@ Check the [Version Specific Upgrade Notes](upgrading/version-specific.md) articl
 
 This section contains the release notes for Umbraco Workflow 13 including all changes for this version.
 
+### [13.0.4](https://github.com/umbraco/Umbraco.Workflow.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F13.0.4) (March 13th 2024)
+
+* Make dashboard, content app and section registration classes public [#56](https://github.com/umbraco/Umbraco.Workflow.Issues/issues/56)
```
Comment (eshanrnh) [inline 13/umbraco-workflow/release-notes.md:22]:
```suggestion
* Make dashboard, content app, and section registration classes public [#56](https://github.com/umbraco/Umbraco.Workflow.Issues/issues/56)
```

**Review by eshanrnh — APPROVED** [review]
Looks perfect, @nathanwoulfe 🙌 Merging it in.

**Pull request merged**